### PR TITLE
Issue 1 user can create an entry

### PR DESCRIPTION
**Pull request by KiraHike**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Kira! Please add a screenshot of your app in desktop view as well

**Comment by KiraHike** [comment]
image

File: js/main.js [hunk 1]
```diff
@@ -1,2 +1,34 @@
...
+  var notesValue = $newEntryForm.elements.notes.value;
+  var imageValue = $newEntryForm.elements.url.value;
+  var entryObj = {
+    URL: imageValue,
+    title: titleValue,
+    notes: notesValue,
+    entryId: data.nextEntryId
+  };
+  data.nextEntryId++;
+  data.entries.push(entryObj);
```
Comment (codingcodymiller) [inline js/main.js]: You want to add to the beginning of the array, not the end. You will need a different method than `push`

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,34 @@
...
+  $image.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $newEntryForm.reset();
+  var dataJSON = JSON.stringify(data);
+  localStorage.setItem('js-local-storage', dataJSON);
```
Comment (codingcodymiller) [inline js/main.js]: This should be done in the `data.js` file. You only need to update localStorage before the page unloads, not after each submit.

File: js/main.js [hunk 3]
```diff
@@ -1,2 +1,34 @@
...
+
+var $newEntryForm = document.querySelector('#entry-form');
+
+function clickSave(event) {
+  event.preventDefault();
+  var titleValue = $newEntryForm.elements.title.value;
+  var notesValue = $newEntryForm.elements.notes.value;
+  var imageValue = $newEntryForm.elements.url.value;
+  var entryObj = {
+    URL: imageValue,
```
Comment (codingcodymiller) [inline js/main.js]: I would keep this lowercase, just `url`

File: index.html [hunk 4]
```diff
@@ -16,9 +16,57 @@
...
+          <div class="column-full">
+            <div class="button-container">
+              <button class="button-save">SAVE</button>
+            </div>
+          </div>
+        </div>
+      </form>
+    </div>
   </main>
+  <footer></footer>
```
Comment (codingcodymiller) [inline index.html:69]: This empty `footer` can be removed. You may adda footer later, but a feature's code should only include code for that specific feature.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! A couple notes for you:

File: index.html (hunk 4 above)
Comment (KiraHike) [reply index.html:69]: Left this in for styling purposes, as discussed.

File: js/main.js (hunk 3 above)
Comment (KiraHike) [reply js/main.js]: Done.

File: js/main.js (hunk 1 above)
Comment (KiraHike) [reply js/main.js]: Changed this to unshift

File: js/main.js (hunk 2 above)
Comment (KiraHike) [reply js/main.js]: Done.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**